### ISSUE TITLE
Баг

### ISSUE THREAD

**Issue by MazerUNKNOW** [Ferrster/Snowrunner-Manual-Gearbox-Mod#19]
А возможно как то исправить или что то сделать с данной проблемой ???
Короче когда играешь на геймпаде то на автомате на первой скорости можно включить дифференциал, так получается когда нажимаешь RB и включаешь дифференциал, то если отпускаешь кнопку RB сразу как бы вырубается дифференциал, но если переключив и держишь кнопку RB то работает корректно, просто на клавиатуре такой проблемы нету, только на геймпаде.
Или если что не могли бы вы передать эту проблему разрабу данного мода, а то мод вообще огонь без него теперь вообще не играю))) но просто напрягает на геймпаде 2 кнопки держать))

**Comment by Ferrster** [comment]
Да, это известная проблема( Пофикшу, когда допишу код для шифтеров. Ну а пока придется страдать, сори
